interface/wx/httprequest.h

```diff
@@ -0,0 +1,232 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        httprequest.h
+// Created:     2018-10-14
+// Copyright:   (c) 2018 wxWidgets development team
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+/**
+    @class wxHTTPRequest
+
+    wxHTTPRequest allows for simple HTTP requests using the operating systems
+    components as implementation. The latest features of their operating
+    system will be used if available.
+
+    This class is well suited for simple HTTP requests.
+
+    For example:
+
+    @code
+        wxHTTPRequest request("http://www.wxwidgets.org/downloads/logos/blocks.png");
```

Comment:
Should use `https://` in the URL just to immediately show that it is supported.

Reply:
Of course, actually started as https but then copied the URL from the downloads page, which should probably link to https now